vizro-ai/docs/index.md

```diff
@@ -47,7 +47,8 @@ If a feature you need for your dashboard isn't currently supported by Vizro-AI y
     [:octicons-arrow-right-24: Create advanced charts](pages/user-guides/create-advanced-charts.md)</br>
     [:octicons-arrow-right-24: Add charts to a dashboard](pages/user-guides/add-generated-chart-usecase.md)</br>
     [:octicons-arrow-right-24: Generate a complex dashboard](pages/user-guides/create-complex-dashboard.md)</br>
-    [:octicons-arrow-right-24: Retrieve code for a generated dashboard](pages/user-guides/retrieve-dashboard-code.md)
+    [:octicons-arrow-right-24: Retrieve code for a generated dashboard](pages/user-guides/retrieve-dashboard-code.md)</br>
```

Comment:
I don't think we should add either of these to the index page right now -- it's really only for calling out a few key pages and doesn't function as a full table of contents. Let's just delete both if that's OK with you.

Reply:
Make sense to me. I dropped "Use Vizro-AI as Langchain tools" and "Generate a complex dashboard" and kept "Retrieve code for a generated dashboard" , as retrieving code is one of the most important features out of the dashboard functionality